account_move_tier_validation/models/tier_definition.py

```diff
@@ -12,3 +12,10 @@ def _get_tier_validation_model_names(self):
         res = super(TierDefinition, self)._get_tier_validation_model_names()
         res.append("account.move")
         return res
+
+    def request_validation(self):
+        super().request_validation()
+        action = self.env.ref(
+            "account_move_tier_validation.invoice_send_validation_request"
+        )
+        return action.read()[0]
```

Comment:
I think we are breaking something here by not returning the expected result:

- https://github.com/OCA/server-ux/blob/6dcf4aee1c177b5a8894173cdb9e1b3dff68c784/base_tier_validation/models/tier_validation.py#L368-L392
- https://github.com/OCA/server-ux/blob/6dcf4aee1c177b5a8894173cdb9e1b3dff68c784/base_tier_validation/models/tier_validation.py#L193
Unless I'm missing something?
@TDu do you know more about this?

Reply:
@sebalix You are correct, something is fishy, there...

Reply:
this was a mistake, override was meant to be on `tier.validation` not `tier.definition`